nautobot/nautobot_data_validation_engine/views.py

```diff
@@ -4,18 +4,20 @@
 from django.contrib.contenttypes.models import ContentType
 from django_tables2 import RequestConfig
 
-from nautobot.apps.views import (
-    ObjectBulkDestroyViewMixin,
-    ObjectDestroyViewMixin,
-    ObjectDetailViewMixin,
-    ObjectListViewMixin,
-)
 from nautobot.core.ui.choices import SectionChoices
 from nautobot.core.ui.object_detail import (
     ObjectDetailContent,
     ObjectFieldsPanel,
 )
 from nautobot.core.views.generic import ObjectView
+from nautobot.core.views.mixins import (
+    ObjectBulkDestroyViewMixin,
+    ObjectChangeLogViewMixin,
+    ObjectDestroyViewMixin,
+    ObjectDetailViewMixin,
+    ObjectListViewMixin,
+    ObjectNotesViewMixin,
+)
```

Comment:
What keeps us from using UIViewSet? Lack of bulk-edit support?

Reply:
Ah okay, I guess DataCompliance isn't user-editable, which makes sense as I think about it. No issue here then.